<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileInformation</class>
 <widget class="QDialog" name="FileInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>488</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>311</width>
    <height>488</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>331</width>
    <height>488</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FileInformation</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{  
background-color: white;  
border: none;
color: red;
text-align: center;
margin: 4px 2px;
opacity: 0.6;
text-decoration: none;
border: 1px solid red;
}
QPushButton:hover
{
	margin: 2px 1px;
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>File name</string>
   </property>
  </widget>
  <widget class="QListWidget" name="usernameList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>311</width>
     <height>281</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget::item
{
    background: rgb(255,255,255); 
	color : rgb(255,255,255); 
}
QListWidget::item:focus
{
	background: rgb(255,255,255); 
	color : rgb(255,255,255); 
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{  
background-color: rgb(0, 173, 0);
border: none;
color: white;
text-align: center;
margin: 4px 2px;
opacity: 0.6;
text-decoration: none;
}

QPushButton:hover
{
	margin: 2px 1px;
}</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 32px;</string>
   </property>
   <property name="text">
    <string>File Information</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteFile">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>311</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{  
background-color: rgb(255, 0, 0);
border: none;
color: white;
text-align: center;
margin: 4px 2px;
opacity: 0.6;
text-decoration: none;
}

QPushButton:hover
{
	margin: 2px 1px;
}</string>
   </property>
   <property name="text">
    <string>Delete file</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
